main.py

```diff
@@ -10,24 +10,17 @@
 # This file contains the startup for the server.
 #
 
-# Begin module imports
-import socket;
-import sys;
-import os;
-# End module imports
-
 # Begin user imports
-#import src.RequestHandler
+from src.server import ChatdServer as Server
 # End user imports
 
 # main
 
-# Create socket
-sock = socket.socket();
-
-# Bind socket to localhost port 
-sock.bind(("localhost",20039));
-
-newsock,_ = sock.accept();
+# Initialize the server
+serv = Server();
 
+# Bind the server
+serv.bind(10001);
```

Comment:
Has the port been cleared? What are you going to do about cross-compatability, or are you just hoping for the best? Is there a way to automatically search and open a port on a server? Having conflicts here would not be good...